examples/jsm/math/Matrix2.js

```diff
@@ -0,0 +1,54 @@
+export class Matrix2 {
```

Comment:
Should we move this class to the core instead?

Reply:
I'll leave that up to someone else to decide. With tree shaking I suppose it's not a big deal to move but I'm not sure of what the exact rules are for what belongs in examples vs core.

Reply:
It's a case by case decision but since `Vector2` is part of the core it seems more consistent to add `Matrix2` as well.

Reply:
Okay - I can move it if you'd like!

Reply:
@WestLangley Are you okay with that?

Reply:
I am. Thanks!

Reply:
Great - I've just moved the files to core